From dd64fd9ef6d176000735d527ee8772666683f80c Mon Sep 17 00:00:00 2001
From: Meng Wei <wei.meng@intel.com>
Date: Mon, 10 Dec 2018 11:09:17 +0800
Subject: [PATCH 0995/1214] media: Disable FW IRQ and response of EOF

Disabling FW IRQ and response of EOF to decrease CPU wake up by 1
for every frame.

Change-Id: I6cbd5d0d097d8c4f2fde8fb9b400e5e6c2f7c228
Tracked-On: PKT-1732
Signed-off-by: Hongyu Yi <hongyu.yi@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu-isys-queue.c | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu-isys-queue.c b/drivers/media/pci/intel/ipu-isys-queue.c
index 2f0db1a..a88c0d7 100644
--- a/drivers/media/pci/intel/ipu-isys-queue.c
+++ b/drivers/media/pci/intel/ipu-isys-queue.c
@@ -488,11 +488,15 @@ void ipu_isys_buffer_list_to_ipu_fw_isys_frame_buff_set(
 
 	set->send_irq_sof = 1;
 	set->send_resp_sof = 1;
-	set->send_irq_eof = 1;
-	set->send_resp_eof = 1;
+
 #if defined(CONFIG_VIDEO_INTEL_IPU4) || defined(CONFIG_VIDEO_INTEL_IPU4P)
 	set->send_irq_capture_ack = 1;
 	set->send_irq_capture_done = 1;
+	set->send_irq_eof = 1;
+	set->send_resp_eof = 1;
+#else
+	set->send_irq_eof = 0;
+	set->send_resp_eof = 0;
 #endif
 
 	list_for_each_entry(ib, &bl->head, head) {
-- 
2.7.4

